build-aux/m4/bitcoin_qt.m4

```diff
@@ -291,7 +273,7 @@ AC_DEFUN([_BITCOIN_QT_CHECK_QT5],[
       #endif
     ]],
     [[
-      #if QT_VERSION < 0x050000
+      #if QT_VERSION < 0x050501
```

Comment:
missing changes for `QTXCBQPA`

Reply:
still missing https://github.com/bitcoin/bitcoin/pull/15393/files#diff-ab26cccb62e7a807f1dce314412a5e17R359